### ISSUE TITLE
Issues when secure transfer is enabled

### ISSUE THREAD

**Issue by dagiro** [Azure/azure-powershell#8797]
`Get-AzureRmHDInsightCluster` and `Get-AzHDInsightCluster` do not return `DefaultStorageAccount` and `DefaultStorageContainer` when secure transfer is enabled on the storage account.

**Comment by vladimir-shcherbakov** [comment]
@dagiro Can you help us to get more context and fill out the `Az module bug report`: 


## Description



## Steps to reproduce

```powershell

```

## Environment data



```

```

## Module versions



```powershell

```

## Debug output



```

```

## Error output



```

```

**Comment by dagiro** [comment]
Create a storage account through portal which will make secure transfer enabled by default. Else just set it to enabled after creation.  Create cluster using that storage account.  Connect, Connect-AzAccount.  Then execute. Get-AzHDInsightCluster -ClusterName <clusterName>. Executes without error.  However, there are no values for DefaultStorageAccount and DefaultStorageContainer.

**Comment by maddieclayton** [comment]
@momusoru @z01c Can you take a look at this issue?

**Comment by dagiro** [comment]
Get-AzHDInsightJobOutput also appears to be affected by secure transfer. Did not test the similar cmdlet from AzureRM - Get-AzureRmHDInsightJobOutput. For repro, I was running the 1st example from https://docs.microsoft.com/en-us/azure/hdinsight/hadoop/apache-hadoop-use-hive-powershell. When secure transfer is enabled on the storage account, I receive the following:
```
Get-AzHDInsightJobOutput : Couldn't find storage information for cluster girouxHadoop2
At line:44 char:1
+ Get-AzHDInsightJobOutput `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzHDInsightJobOutput], CloudException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.HDInsight.GetAzureHDInsightJobOutputCommand
```

**Comment by dagiro** [comment]
Also add `Invoke-AzHDInsightHiveJob` to the list.  For repro steps, followed 2nd PowerShell example in the same article referenced earlier. Did not test `Invoke-AzureHDInsightHiveJob`.

**Comment by dagiro** [comment]
In [Manage Apache Hadoop clusters in HDInsight by using Azure PowerShell](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-administer-use-powershell#find-the-default-storage-account) I have added an [!IMPORTANT] stating, "The values for `DefaultStorageAccount`, and `DefaultStorageContainer` are not returned from `Get-AzHDInsightCluster` when secure transfer is enabled on the storage account." I plan to add this language to other related articles. Please let me know if you prefer different language.

**Comment by idear1203** [comment]
@dagiro, thank you for the notification. This issue has been fixed in the recent release. Let us know if the issue still exists.

**Comment by idear1203** [comment]
This is fixed in the PR: [#9069 [HDInsight] - Add a cmdlet to update gateway credential](https://github.com/Azure/azure-powershell/pull/9069)
